network/src/commonMain/kotlin/com/wire/kalium/network/api/RefreshToken.kt

```diff
@@ -0,0 +1,5 @@
+package com.wire.kalium.network.api
+
+object RefreshTokenProperties {
+    const val COOKIE_NAME = "zuid"
```

Comment:
Shouldn't the key value still be `cookie` according to the BE docs?

<img width="972" alt="Screenshot 2021-12-17 at 16 17 57" src="https://user-images.githubusercontent.com/2468164/146566444-a0d8f67b-2f9a-48d4-bfc1-5d097d5b58cd.png">

Reply:
The the cookie we need to read when receiving the credentials is called `zuid`. So the name here is for unpacking the cookie from the response.

If we want to use the cookie later for some requests, it's called `cookie` 🤷🏼‍♂️